estuary_updater/handlers/koji.py

```diff
@@ -37,17 +41,21 @@ def handle(self, msg):
         """
         topic = msg['topic']
 
-        if topic == '/topic/VirtualTopic.eng.brew.build.complete':
-            self.build_complete_handler(msg)
+        if topic == '/topic/VirtualTopic.eng.brew.build.complete' or \
+                topic == '/topic/VirtualTopic.eng.brew.build.building' or \
+                topic == '/topic/VirtualTopic.eng.brew.build.failed' or \
+                topic == '/topic/VirtualTopic.eng.brew.build.canceled' or \
+                topic == '/topic/VirtualTopic.eng.brew.build.deleted':
+            self.build_handler(msg)
         elif topic == '/topic/VirtualTopic.eng.brew.build.tag' or \
                 topic == '/topic/VirtualTopic.eng.brew.build.untag':
             self.build_tag_handler(msg)
         else:
             raise RuntimeError('This message is unable to be handled: {0}'.format(msg))
 
-    def build_complete_handler(self, msg):
+    def build_handler(self, msg):
         """
-        Handle a build complete message and update Neo4j if necessary.
+        Handle a build building, failed, and complete message and update Neo4j if necessary.
```

Comment:
How about the following?
`Handle a build state message and update Neo4j if necessary.`